### PR TITLE
TELCODOCS-1822: Use Cases - module.name not module-name

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
D/S Docs: [MGMT-17316] Use Cases - module.name not module-name

Jira: https://issues.redhat.com/browse/TELCODOCS-1822

Version(s): openshift-4.16.0, openshift-4.15.0, openshift-4.14.0

Link to docs preview: https://75299--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management.html#kmm-customizing-upgrades-for-kernel-modules_kernel-module-management-operator

SME review: @qbarrand
QE review: @cdvultur

### CODE REVIEW

File: modules/kmm-customizing-upgrades-for-kernel-modules.adoc [hunk 1]
```diff
@@ -46,7 +46,7 @@ $ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namesp
 
 . If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
 +
-If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module.namespace>.<module.name>` label value to the new `$moduleVersion` as set in the `Module`.
```
Comment (StephenJamesSmith) [inline modules/kmm-customizing-upgrades-for-kernel-modules.adoc:49]: @cdvultur These are the only instance I found. But, as we discussed, there are instances similar to these in this and other sections, such as: oc label node/<node_name> kmm.node.kubernetes.io/version-module.**<module_namespace>.<module_name>**=<desired_version> . Should we be looking at these?

Comment (StephenJamesSmith) [reply modules/kmm-customizing-upgrades-for-kernel-modules.adoc:49]: @cdvultur We will not use periods here, we will use underscores to conform to style guide.

File: modules/kmm-customizing-upgrades-for-kernel-modules.adoc [hunk 2]
```diff
@@ -46,7 +45,7 @@ $ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namesp
 
 . If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
 +
-If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>` label value to the new `$moduleVersion` as set in the `Module`.
```
Comment (StephenJamesSmith) [inline modules/kmm-customizing-upgrades-for-kernel-modules.adoc:48]: @cdvultur As per our chat, replaced periods and dashes with underscores in the user-replaceable values. If this looks good, please /lgtm. Thx!

**Review by cdvultur — APPROVED** [review]
/lgtm

File: modules/kmm-customizing-upgrades-for-kernel-modules.adoc (hunk 2 above)
Comment (dfitzmau) [reply modules/kmm-customizing-upgrades-for-kernel-modules.adoc:48]: For step 4:

`$ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-`

Hyphen at end is OK?

**Comment by dfitzmau** [comment]
Hi @StephenJamesSmith .

I added some minor (maybe unrelated) comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

File: modules/kmm-customizing-upgrades-for-kernel-modules.adoc (hunk 2 above)
Comment (StephenJamesSmith) [reply modules/kmm-customizing-upgrades-for-kernel-modules.adoc:48]: Hyphen is ok.

Comment (StephenJamesSmith) [reply modules/kmm-customizing-upgrades-for-kernel-modules.adoc:48]: Mentioning the steps is very helpful to users. No plans on updating the steps, but if we do, will update this accordingly.

**Review by jldohmann — APPROVED** [review]
LGTM

**Comment by dfitzmau** [comment]
> oop @StephenJamesSmith I just noticed your commits, can you squash them before merge? thanks 🙏

Nicely spotted!

**Pull request merged**